crates/rattler_conda_types/src/package/paths.rs

```diff
@@ -166,7 +166,7 @@ pub struct PathsEntry {
     pub path_type: PathType,
 
     /// The type of the file, either binary or text.
-    #[serde(default, skip_serializing_if = "FileMode::is_binary")]
+    // #[serde(default, skip_serializing_if = "FileMode::is_binary")]
```

Comment:
Should we at least keep the default in, for the cases where it is not mentioned?